ConsentViewController/Classes/Views/tvOS/NativePrivacyManager/CCPA/SPCCPANativePrivacyManagerViewController.swift

```diff
@@ -31,21 +31,24 @@ import UIKit
     @IBOutlet var rejectButton: SPAppleTVButton!
     @IBOutlet var saveAndExitButton: SPAppleTVButton!
     @IBOutlet var privacyPolicyButton: SPAppleTVButton!
+    @IBOutlet var customViewButton: SPAppleTVButton!
     @IBOutlet var doNotSellTableView: UITableView!
     @IBOutlet var actionsContainer: UIStackView!
 
     override func viewDidLoad() {
         super.viewDidLoad()
         initConsentsSnapshot()
         setHeader()
+        let showBtn = pmData.rootView.settingsRoot
         loadTextView(forComponentId: "PublisherDescription", textView: descriptionTextView, bounces: false)
         descriptionTextView.flashScrollIndicators()
-        loadButton(forComponentId: "AcceptAllButton", button: acceptButton)
-        loadButton(forComponentId: "RejectAllButton", button: rejectButton)
-        loadButton(forComponentId: "SaveAndExitButton", button: saveAndExitButton)
-        loadButton(forComponentId: "NavCategoriesButton", button: managePreferenceButton)
-        loadButton(forComponentId: "NavVendorsButton", button: ourPartners)
-        loadButton(forComponentId: "NavPrivacyPolicyButton", button: privacyPolicyButton)
+        loadButton(forComponentId: "AcceptAllButton", button: acceptButton, show: showBtn.showAcceptAllBtn)
```

Comment:
@Nevazhnovu Do we really need to rely on the those flags to determine whether to show a button or not?
Can't we decide to show/hide a button based on its presence in the payload instead of the flag?